var/spack/repos/builtin/packages/clhep/package.py

```diff
@@ -26,13 +26,23 @@
 from spack import *
 
 
-class Clhep(Package):
+class Clhep(CMakePackage):
     """CLHEP is a C++ Class Library for High Energy Physics. """
     homepage = "http://proj-clhep.web.cern.ch/proj-clhep/"
     url      = "http://proj-clhep.web.cern.ch/proj-clhep/DISTRIBUTION/tarFiles/clhep-2.2.0.5.tgz"
-    list_url = "https://proj-clhep.web.cern.ch/proj-clhep/DISTRIBUTION/"
+    list_url = "https://proj-clhep.web.cern.ch/proj-clhep/"
+    list_depth = 3
```

Comment:
I just checked and I think a `list_depth` of 2 is sufficient. 1 works if you don't care about the 1.8 releases.